packages/api-client/src/components/Server/ServerVariablesForm.vue

```diff
@@ -22,19 +28,17 @@ function setVariable(name: string, value: string) {
 }
 
 const getVariable = (name: string) => {
-  return (
-    props.values?.[name] ??
-    props.variables?.[name]?.default ??
-    ''
-  ).toString()
+  return (values?.[name] ?? variables?.[name]?.default ?? '').toString()
```

Comment:
This should be the other way around? obecause it theres a default then this will be 
`{ default: 'xxxx' }`.toString()

how about
```suggestion
  return (values?.[name]?.default ?? variables?.[name] ?? 'variable').toString()
```
threw an optional default name in there, though it should never get hit

Reply:
that's how it's been i just destructured prop here, but i believe you are right!

Reply:
yup it was like that before, but wanna fix it anyway 😄

Reply:
we can just commit this suggestion actually

Reply:
Whoops  I misread it LOL my eyes! This is why I gotta review in the editor lol